src/QMCHamiltonians/ECPComponentBuilder.2.cpp

```diff
@@ -103,7 +103,12 @@ void ECPComponentBuilder::buildSemiLocalAndLocal(std::vector<xmlNodePtr>& semiPt
       aAttrib.add(lstr, "l");
       aAttrib.add(rc, "cutoff");
       aAttrib.put(cur_vps);
-      rmax  = std::max(rmax, rc);
+      rmax = std::max(rmax, rc);
+      if (angMon.find(lstr) == angMon.end())
+      {
+        app_error() << "requested angular momentum " << lstr << " not available.\n";
```

Comment:
examples
https://github.com/QMCPACK/qmcpack/blob/34e5dc1d700d58150759bcc8bdf8aa0dfa03bf4b/src/QMCWaveFunctions/LCAO/LCAOrbitalBuilder.cpp#L469
You may use ostringstream to buffer contents if it is long.
https://github.com/QMCPACK/qmcpack/blob/34e5dc1d700d58150759bcc8bdf8aa0dfa03bf4b/src/QMCWaveFunctions/LCAO/LCAOrbitalBuilder.cpp#L702

Reply:
fixed